cmd/cstor-volume-grpc/app/command/commands.go

```diff
@@ -0,0 +1,280 @@
+/*
```

Comment:
what is the purpose of the command file? Who uses this?

Reply:
@Moteesh we need to think of a proper place for commands. If this is only used for testing purposes, then can we make use of _test.go file? It need not be a cli always?

Reply:
this command file has both server and client commands namely : 
server start , 
client snapshot create, 
client snapshot delete
@AmitKumarDas , I have implemented both server and client in the same binary. client commands invoke the local grpc server. this is useful as a local testing environment also without the need for another binary to test.
Is it possible to use _test.go files for real time testing ?